api/server/server_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 func testServer(ds models.Datastore, mq models.MessageQueue, logDB models.LogStore, rnr agent.Agent, nodeType NodeType, opts ...Option) *Server {
 	return New(context.Background(), append(opts,
-		WithLogLevel(getEnv(EnvLogLevel, DefaultLogLevel)),
+		WithLogLevel("debug"),
```

Comment:
Not sure about this. Looks like a dev time thing that shouldn't be commited.

Reply:
just looks like the kind of thing that someone might be using somewhere, via env vars and this might upset them.

Reply:
just logs yo